storyruntime/Kubernetes.py

```diff
@@ -576,10 +580,33 @@ def get_liveness_probe(cls, app, service: str):
         service_tag_uuids = await Database.get_service_tag_uuids(
             app.config, [{'service_uuid': service_uuid, 'tag': tag}]
         )
+
         assert len(service_tag_uuids) == 1
-        limits = await Database.get_service_limits(
-            app.config, service_tag_uuids[0]
-        )
+
+        # Set the limits back to the defaults if we're updating
+        # a deployment, or the image_pull_policy is Always.
+        if update_deployment or app.image_pull_policy() == 'Always':
+            # let us check the memory
+            limits = {
+                'cpu': ServiceConstants.default_cpu,
+                'memory': ServiceConstants.default_memory  # 200Mi or 210MB
```

Comment:
```suggestion
                'memory': ServiceConstants.default_memory
```

Constants will change.